scripts/typecheck_tests.py

```diff
@@ -7,9 +7,13 @@
 from distutils import dir_util, spawn
 from typing import Dict, List, Pattern, Union
 
-from scripts.git_helpers import checkout_target_tag
+from scripts.git_helpers import git_checkout_drf
 from scripts.paths import DRF_SOURCE_DIRECTORY, PROJECT_DIRECTORY, STUBS_DIRECTORY
 
+# django-rest-framework commit hash to check against (ignored with --drf_version)
+# This should be updated periodically or after every DRF release.
+DRF_GIT_REF = "22d206c1e0dbc03840c4d190f7eda537c0f2010a"
```

Comment:
LGTM, this is 1 week old: https://github.com/encode/django-rest-framework/commit/22d206c1e0dbc03840c4d190f7eda537c0f2010a